semgrep-core/src/matching/Generic_vs_generic.ml

```diff
@@ -458,25 +461,34 @@ let rec m_name a b =
                };
              _;
            } as infob) ) ) -> (
+      let is_resolved =
+        match resolved with
+        | B.ResolvedName _
+        | B.ImportedEntity _
```

Comment:
Great! But we need a test here, at least https://semgrep.dev/s/qzP1, and better if we could come up with some other.